src/rocm_docs/__init__.py

```diff
@@ -117,7 +117,7 @@ def run_doxygen(
         self._ran_doxygen = True
         doxygen_root = self.to_path(doxygen_root)
         if doxygen_root is None:
-            doxygen_root = Path("docs/doxygen")
+            doxygen_root = Path("./doxygen")
```

Comment:
One more change, please change the doxygen folder to .doxygen. As it is a configuration folder, let's differentiate from the other folders.

Reply:
Done with commit [b6cfe02](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/9/commits/b6cfe0201a4506e87b928dd4eca1094030735ad3)

Also tested with rtd branch for hipBLAS
https://github.com/ROCmSoftwarePlatform/hipBLAS/tree/rtd